x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/index.test.tsx

```diff
@@ -46,6 +46,7 @@ const TestComponent = (props: TestComponentProps) => {
     <TestProviders store={store}>
       <TimelineDataTable
         columns={initialEnrichedColumns}
+        columnIds={initialEnrichedColumns.map((c) => c.id)}
```

Comment:
Can you pull this out into a `useMemo`?

Reply:
actually, this is a test...ignore me 😅

Reply:
hehe 😆. Still corrected here: 886b161ef0e